src/styles/blocks/form-field.scss

```diff
@@ -0,0 +1,33 @@
+.form-field {
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  height: 48px;
+  padding: 14px 16px;
+
+  border: 1px solid transparent;
+  border-radius: 40px;
+  background-color: #f8f8fa;
```

Comment:
Checklist 12th item. The speed of animations is the same throughout the landing page (for example, increasing when hovering or moving blocks when scrolling). You should add transition for all the hovers, too